src/rust/engine/fs/src/lib.rs

```diff
@@ -86,6 +89,56 @@ pub fn default_config_path() -> PathBuf {
   config_path.join("pants")
 }
 
+#[derive(Derivative, Clone, Debug, Eq)]
+#[derivative(PartialEq, Hash)]
+pub struct RelativePath(PathBuf);
```

Comment:
This was just a move from src/rust/engine/process_execution/src along with tests.rs below.